website/content/docs/release-notes/v0_18_0.mdx

```diff
@@ -228,5 +240,24 @@ description: >-
     </td>
   </tr>
 
+  <tr>
+    <td style={{verticalAlign: 'middle'}}>
+    0.18.x
+    </td>
+    <td style={{verticalAlign: 'middle'}}>
+    Boundary version 0.18.x controllers or workers are unable to establish SSH connections using the <code>boundary connect ssh</code> command
```

Comment:
I think we've only had reports of this happening with the CLI, but in theory it could happen on any VPN network boundary.
```suggestion
    Boundary version 0.18.x CLI is unable to establish connections using the <code>boundary connect</code> command.
```